frontend/src/pages/Admin/components/CurrentPermissions.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function CurrentPermissions({ regions, scope }) {
+  const regionsStr = regions.length === 1 ? 'Region' : 'Regions';
+  const regionMsg = `${regionsStr} ${regions.join(', ')}`;
+  return (
+    <li key={scope}>
```

Comment:
I don't believe the `key` is needed here, it would be needed on the `<CurrentPermissions>` component, most likely.